pkg/standalone/standalone.go

```diff
@@ -51,6 +51,13 @@ const (
 	redisDockerImageName       = "redislabs/rejson"
 	zipkinDockerImageName      = "openzipkin/zipkin"
 
+	githubContainerRegistryName = "ghcr"
+	ghcrURL                     = "ghcr.io/dapr"
+	daprGhcrImageName           = "dapr"
+	dockerContainerRegistryName = "dockerhub"
+	redisGhcrImageName          = "3rdparty/redis"
+	zipkinGhcrImageName         = "3rdparty/zipkin"
```

Comment:
Just a thought shouldn't we retain the upstream in our paths,
For example instead of `3rdparty/zipkin`, how about `3rdparty/openzipkin/zipkin`?

Reply:
Agreed, I have similar understanding, to specifically mention which particular flavor is getting used. In future, it can also result in using different flavors, if required.